tools/integration_tests/util/setup/setup.go

```diff
@@ -362,7 +362,7 @@ func CleanUpDir(directoryPath string) {
 
 	for _, d := range dir {
 		err := os.RemoveAll(path.Join([]string{directoryPath, d.Name()}...))
-		if err != nil {
+		if err != nil && !strings.Contains(err.Error(), "no such file or directory") {
```

Comment:
Why was this change required?

Reply:
In Delete tests of admin permissions with managed folder is already deleting `ManagedFolder1` and in teardown we again try to clean the same in case of View permission on bucket.
https://screenshot.googleplex.com/AJdfFn38mGZ2Ptp